### DISCUSSION TITLE
Installation failure on MacOS 12, Intel based machine

### DISCUSSION THREAD

**Discussion by sq3101** [su2code/SU2#1779] [Q&A] [answered]
Hi All,

Has anybody met this issue before - that compiling SU2 code from source with ninja, the final step of install comes with failure (showed in attached screenshot) and quit from installation. Can anyone bring some assistance to this problem?

<img width="422" alt="截屏2022-10-04 14 28 34" src="https://user-images.githubusercontent.com/62012004/193728884-9460755c-c1fa-4eb6-9872-b74f9209978d.png">

Kind regards
Shuang

**Comment by NAnand-TUD** [comment, marked as answer]
Hello, I think the best would be to get a new code and start the installation. In my experience, when you change to branches the externals lose track of the git hashes.

**Reply by sq3101** [reply]
Hi, thanks for the reply. I've tried with new code several times from 7.3.1 to 7.4.0. This problem do exist. I'm not sure why this would happen on my Mac, but the code installation on linux goes well.

**Reply by NAnand-TUD** [reply]
Well, I just reinstalled SU2 on my Mac and I can confirm everything went well. I hope you are following the procedure mentioned on [SU2-GitHubPages](https://su2code.github.io/docs_v7/Build-SU2-Linux-MacOS/).

Hope this fixes the issue.

**Reply by sq3101** [reply]
Thanks! I've re-installed using terminal in VScode and the code has been successfully installed. Looks like there are some problems in my system-native terminal. So please don't worry about my problem.
